apps/protocol-api/src/prisma/resolvers/User.ts

```diff
@@ -127,4 +148,20 @@ export class UserCustomResolver {
       where: { address: { equals: address } },
     });
   }
+
+  @TypeGraphQL.Query((_returns) => Int, { nullable: false })
+  async getContributionCountForUser(
+    @TypeGraphQL.Ctx() ctx: Context,
+    @TypeGraphQL.Args() args: GetUserContributionCountArgs
+  ) {
+    return await getPrismaFromContext(ctx).contribution.count({
```

Comment:
I think a group would work better here. This, as far as I can tell, provides a single count, but I think the client is going to want to get the counts for multiple days or weeks in a single call because I believe we will use these to populate a contribution calendar and bar chart.


Example taken from .prisma/client/index.d.ts
```
    /**
     * Group by Contribution.
     * Note, that providing `undefined` is treated as the value not being there.
     * Read more here: https://pris.ly/d/null-undefined
     * @param {ContributionGroupByArgs} args - Group by arguments.
     * @example
     * // Group by city, order by createdAt, get count
     * const result = await prisma.user.groupBy({
     *   by: ['city', 'createdAt'],
     *   orderBy: {
     *     createdAt: true
     *   },
     *   _count: {
     *     _all: true
     *   },
     * })
     * 
    **/

```

Reply:
https://github.com/prisma/prisma/discussions/11692

Reply:
Probably has to be done with a raw sql query